example/cordova/www/pages/metrics.html

```diff
@@ -45,20 +45,31 @@ <h4>Response:</h4>
   return {
     beforeCreate() {
       var self = this;
-      self.$("#metrics-info").html("No metrics");
-      window.MobileCore.MetricsService.getMetrics(
-        function (metrics) {
+      self.$("#metrics-info").html("Fetching metrics...");
+
+      self.metricsService = new aerogear.MetricsService({ url: "http://192.168.1.6:3000/metrics" });
```

Comment:
However this example works it is not what other SDK is doing. Action is that Metrics are instantiated and are sending data to server on start.